Sources/Public/iOS/AnimatedControl.swift

```diff
@@ -126,6 +126,11 @@ open class AnimatedControl: UIControl {
     get { animationView.animationSpeed }
   }
 
+  public override var accessibilityTraits: UIAccessibilityTraits {
+    set { super.accessibilityTraits = newValue }
+    get { super.accessibilityTraits.union(.button) }
```

Comment:
@brynbodayle Are all controls buttons? I'm worried that with things like `UITextField` subclassing `UIControl` this may not be a valid assumption at the root `AnimatedControl` and instead should be something we add on a per-subclass basis

Reply:
Hmm, I think it in this case all our controls would have user interaction, but in the case of a slider it wouldn't be a button. Let me update this to only be button for `AnimatedButton` and `AnimatedSwitch`